src/test/suite/install.test.ts

```diff
@@ -54,29 +54,26 @@ suite("ensure install methods are functioning as expected", function() {
 		installKubectlSpy.resetHistory();	
 	});
 
-	var testVar = test("ensure cli version checking works correctly", async function() {
-		if(!process.env.VSCODE_CAMELK_GITHUB_TOKEN) {
-			testVar.skip();
-		}
+	const testInvalidVersionVar = test("ensure cli version checking works correctly: detect an update needed with invalid version specified", async function() {
+		skipIfNoGithubTokenAvailable(testInvalidVersionVar);
 
-		await versionUtils.checkKamelNeedsUpdate('bogusversion').then( () => {
-			assert.fail('cli version checking, negative case worked');
-		}).catch( (error) => {
-			assert.ok(error, 'cli version checking, negative case worked');
-		});
+		const needsUpdate: boolean = await versionUtils.checkKamelNeedsUpdate('bogusversion');
+		assert.isTrue(needsUpdate, 'It is mentioned that there is no need of update even with an invalid kamel version');
+	});
```

Comment:
argh, the test wasn't checking that. the checkKamelNeedsUpdate seems to not care of the version provided https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729244531#L706-L711

Reply:
fixed:
- without update of Camel K defautl version: https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729248437#L603 
- with update of Camel K default version https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729248668#L607